doc/guides/technical-values.md

```diff
@@ -0,0 +1,66 @@
+# Technical values and priorities
+
+The project uses these technical values to establish priorities and guide
+collaboration.
+
+## Values and priority level
+
+This is only a subset of the overall priorities, listing only
+those which are at the top of the priority list. Anything on this
+list is very important for the project despite the relative priorities shown.
+
+* Priority 1 - Developer experience
+* Priority 2 - Stability
+* Priority 3 - Operational qualities
+* Priority 4 - Node.js maintainer experience
+* Priority 5 - Technology and API currency
+
+## Value descriptions
+
+### 1 - Developer experience
+We value ensuring that developers are productive and enjoy developing
+with Node.js. Some key elements of this include:
+* Approachability (both technical and community)
+* Great documentation
+* Bundling friction-reducing APIs and components, even though
+  they could be provided externally
+* Enabling/supporting external packages to ensure overall developer experience
+
+### 2 - Stability
+Whenever possible, we seek to insure that currently-working code continues to
+work. We want to earn the trust of developers and end-users. Therefore, we value
+stability.
+Some key elements of this include:
+* Backwards compatibility
+* Predictable and stable releases
+* A strong safety net, including testing how changes
+  in Node.js affect popular packages
+* Careful consideration of what goes into long term support (LTS) releases
+
+### 3 - Operational qualities
+We value keeping Node.js safe, performant, and lightweight.
+We value enabling the ability to investigate and debug problems in
+development and production. Some key elements of this include:
+* High throughput (speed)
+* Fast startup
+* Small binary size
+* Small memory footprint
+* High-quality debugging tools
+* Diagnostic tooling (profilers, heapdumps, coredumps, etc.)
```

Comment:
I don't think of heapdumps and coredumps as tools. They're data consumed by tools like profilers and debuggers. Maybe this?:

```suggestion
* Robust diagnostic tools (profilers, etc.)
```